### ISSUE TITLE
[Adaptive Cards - Designer-ELEMENT PROPERTIES] - Screen reader is incorrectly announcing the position information for the menu items of all the expandable control present inside the 'ELEMENT PROPERTIES' section on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9313]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Screen Reader: Narrator, NVDA

**Repro Steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Designer' control present on the header section of the page using tab key and activate using enter key.
4. Navigate to the 'ELEMENT PROPERTIES' using tab key and activate using enter key.
5. Navigate to all the expandable control on the page using tab key and activate using enter key.
6. Navigate to all the menu items using down arrow key.
7. Verify that, screen reader is announcing the position information for the menu items of all the expandable control present inside the 'ELEMENT PROPERTIES' section on the page or not.

**Actual Result:**
Screen reader is incorrectly announcing the position information for the menu items of all the expandable control present inside the 'ELEMENT PROPERTIES' section on the page. For example, if there are four menu items then screen reader is announcing "1 of 5".

**Observation:**
Same issue replicates with NVDA

**Expected Result:**
Screen reader should properly announce the position information for the menu items of all the expandable control present inside the 'ELEMENT PROPERTIES' section on the page.  For example, if there are four menu items then screen reader should announce as "1 of 4".

**User Experience:**
Visually impaired people who rely on screen readers will be impacted if the position information is announced incorrectly. They may not know how many menu items are available or where they are within the list, making navigation confusing. This can cause users to lose track of their place and make it difficult to review or locate on the page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/24d81f16-fba6-465a-8208-d442eba134ea

### Card JSON

```JSON
NA
```
